### DISCUSSION TITLE
Permission denied (publickey)

### DISCUSSION THREAD

**Discussion by akwagalakwe** [community/community#36981] [Other Feature Feedback, Questions, & Ideas]
Hello,

I have been using github for my class for two months now. Recently, I ran into this error while trying to push my work. "git@github.com: Permission denied (publickey). fatal: Could not read from remote repository.

I tried a couple of online suggestions, but they didn't seem to work. Any thoughts?

**Comment by FatChicken277** [comment]
"GitHub isn't able to authenticate you. So, either you aren't setup with an SSH key, because you haven't [set one up on your machine](https://help.github.com/articles/generating-ssh-keys), or your key isn't associated with your GitHub account."

so try to check your ssh keys or try to change the auth method to https/url instead. [more info](https://docs.github.com/en/get-started/quickstart/set-up-git#authenticating-with-github-from-git)

**Comment by ossanna16** [comment]
Hi there @akwagalakwe and welcome to our community 😄. Thanks so much for asking a great question!
